src/OrchardCore.Modules/OrchardCore.Lists/Liquid/ListItemsFilter.cs

```diff
@@ -29,10 +37,36 @@ public async ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArgument
             {
                 listContentItemId = input.ToStringValue();
             }
+            bool useOrder = false;
```

Comment:
Would be so much simpler with just the `bool order` property, can be try by default. I assume that if Order is disabled on the ListPart then it's not an issue to order on the field? So we don't have to check for the default setting. Or just use what the filter argument states if defined (or true if not defined)